node_response.py

```diff
@@ -0,0 +1,13 @@
+class NodeResponse:
+    def __init__(self, id, attribute_name, attribute_value, votes):
+        self.id = id
+        self.attribute_name = attribute_name
+        self.attribute_value = attribute_value
+        self.votes = votes
+
+    def __str__(self):
```

Comment:
`_str_` method should create a string representation of the object and not print it. Do we need the method at all if we use GTest?